src/main/java/com/online/bookstore/service/BookService.java

```diff
@@ -17,6 +19,8 @@ public interface BookService {
 
     BookDto getBookById(final Long id);
 
+    List<BookDto> getBooksByIds(final Collection<Long> bookIds);
```

Comment:
Why can't we pass just List instead of Collection?

Reply:
to make it more flexible, each type of long collection can be passed here